src/components/ComposedModal/ComposedModal.jsx

```diff
@@ -225,14 +162,15 @@ class ComposedModal extends React.Component {
     return isFetchingData ? (
       <Loading />
     ) : (
-      <StyledModal
+      <CarbonComposedModal
         {...props}
         open={open}
         onClose={this.doNotClose}
-        className={classnames(className, {
+        className={`${classnames(className, {
           'error-modal': type === 'warn',
           'big-modal': isLarge,
         })}
+          ${iotPrefix}--composed-modal--modal`}
```

Comment:
can probably drop the second modal here. 
```suggestion
          ${iotPrefix}--composed-modal`}
```